### PR TITLE
Added the KBaseStructure structure types

### PR DESCRIPTION
**Pull request by qzzhang**

### CODE REVIEW

File: types/KBaseStructure.ProteinStructures/display.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#
+# Define basic display information
+#
+name     : ProteinStructures
+subtitle : Lists of Protein Structures derived from pdb model and/or experimental files
```
Comment (briehl) [inline types/KBaseStructure.ProteinStructures/display.yaml:5]: Is this a single list? Or multiple lists? It maybe should read:
`List of Protein Structures derived from PDB model and/or experimental files`

File: types/KBaseStructure.ProteinStructures/display.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+tooltip  : Lists of Protein Structures derived from pdb model and/or experimental files
+
+icon : none
+
+description : |
+    <p>Lists of Protein Structure derived from pdb model and/or experimental files</p>
```
Comment (briehl) [inline types/KBaseStructure.ProteinStructures/display.yaml:11]: Same here, `List of Protein Structures....`

File: types/KBaseStructure.ExperimentalProteinStructure/spec.json [hunk 3]
```diff
@@ -1,10 +1,12 @@
 {
     "ver":"0.1.0",
     "view_method_ids":[
+        "NarrativeViewers/view_exp_proteinstructure"
```
Comment (briehl) [inline types/KBaseStructure.ExperimentalProteinStructure/spec.json:4]: Note that this function doesn't exist yet.

File: types/KBaseStructure.ModelProteinStructure/spec.json [hunk 4]
```diff
@@ -1,10 +1,12 @@
 {
     "ver":"0.1.0",
     "view_method_ids":[
+        "NarrativeViewers/view_model_proteinstructure"
```
Comment (briehl) [inline types/KBaseStructure.ModelProteinStructure/spec.json:4]: Note that this function doesn't exist yet, either.

**Review by briehl — CHANGES_REQUESTED** [review]
A couple minor comments and text change suggestion.

File: types/KBaseStructure.ProteinStructures/display.yaml (hunk 2 above)
Comment (qzzhang) [reply types/KBaseStructure.ProteinStructures/display.yaml:11]: Same explanation as for the previous one :)

File: types/KBaseStructure.ExperimentalProteinStructure/spec.json (hunk 3 above)
Comment (qzzhang) [reply types/KBaseStructure.ExperimentalProteinStructure/spec.json:4]: For this one and the next, I may remove them for now.  It's a developing/evolving process, Chris leans toward merging these two into one.

**Review by briehl — APPROVED** [review]

**Pull request merged**